src/rdkafka_proto.h

```diff
@@ -597,19 +597,22 @@ typedef struct rd_kafka_Uuid_s {
                 0, 1, ""                                                       \
         }
 
-/**
- * Initialize given UUID to zero UUID.
- *
- * @param uuid UUID to initialize.
- */
-static RD_INLINE RD_UNUSED void rd_kafka_Uuid_init(rd_kafka_Uuid_t *uuid) {
-        memset(uuid, 0, sizeof(*uuid));
-}
-
 static RD_INLINE RD_UNUSED int rd_kafka_Uuid_cmp(rd_kafka_Uuid_t a,
```

Comment:
We should change the signature of this function to use pointer instead when we move these functions to `rdkafka_uuid.c`

Reply:
We took the decision to store and pass Uuid as structs internally, to avoid many heap allocations. Because they're small and not going to change, there's a different cmp function inserted later that takes pointers for when it's needed, like when they're in a list.